.travis.yml

```diff
@@ -1,41 +1,65 @@
 sudo: required
-# dist: xenial
-
+dist: xenial
 
 language: python
 
 matrix:
   # Do not wait for the allowed_failures entry to finish before
   # setting the status
   fast_finish: true
+
   allow_failures:
-    # allow_failures seems to be keyed on the python version
-    - python: 2.7
+  # allow_failures keyed to python 3.5 & skipping tests.
+    - python: "3.5"
+      env: DISTRIB="travisci" PYTHON_VERSION="3.5" FLAKE8_VERSION="*" SKIP_TESTS="true"
+    - python: "3.7"
+      env: DISTRIB="travisci" PIP_FLAGS="--pre" COVERAGE="true"
+           NUMPY_VERSION="*" SCIPY_VERSION="*" PANDAS_VERSION="*"
+           SCIKIT_LEARN_VERSION="*" JOBLIB_VERSION="*" LXML_VERSION="*"
```

Comment:
I guess that this is copy-pasted from nilearn ?

Reply:
Yes, it was.